src/workers/_inputs.js

```diff
@@ -344,8 +411,8 @@ export function unserialize(saved) {
 export function fetchCountMatrix() {
     if ("reloaded" in cache) {
         if (parameters.type == "MatrixMarket") {
-            loadMatrixMarketRaw(parameters.files); 
-        } else {
+            loadMatrixMarketRaw(parameters.files);
+        } else if (parameters.type == "HDF5") {
             loadHDF5Raw(parameters.files);
         }
```

Comment:
Suggest adding an explicit `else` case with an error. This gives a fast fail in case we add a new type and forget to extend this method.